src/webapp_vite/src/App.test.tsx

```diff
@@ -0,0 +1,18 @@
+import { describe, it, vi } from "vitest";
+import { render, screen } from "@testing-library/react";
+import App from "./App";
+
+vi.mock("./ArmView/Arm3d", () => {
```

Comment:
I tried installing node-canvas but threejs still fails on getting a WebGL context.  I don't think it's possible to test Arm3d :(